python/foglamp/services/south/server.py

```diff
@@ -29,13 +29,7 @@ class Server(FoglampMicroservice):
     """" Implements the South Microservice """
 
     # Configuration handled through the Configuration Manager
-    _DEFAULT_CONFIG = {
-        'management_host': {
-            'description': 'Management host',
-            'type': 'string',
-            'default': '127.0.0.1',
-        }
-    }
+    _DEFAULT_CONFIG = {}
```

Comment:
Please add a docstring, how  microservice .py + plugin config will be merged to it

Reply:
It is known that keep_original_items = True merges the configuration.

Reply:
done